mypy/main.py

```diff
@@ -520,6 +520,10 @@ def add_invertible_flag(
         stdout=stdout,
     )
 
+    general_group.add_argument(
+        "-O", "--output", metavar="FORMAT", help="Set a custom output format"
```

Comment:
This should have fixed `choices` list. So that it will error on typo, instead of silently defaulting to standard formatting.

Reply:
Done.